backend/apps/punishment_system/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+# Create your models here.
+class PunishmentType(models.Model):
+    name = models.CharField(max_length=50)
+    value = models.DecimalField(max_digits=6, decimal_places=2, default=0)
+    #image = models.ImageField(upload_to="upload/")
+    def __str__(self):
+        return f'Name: {self.name} Value: {self.value}'
+
+class VineyardGroup(models.Model):
+    name = models.CharField(max_length=30, unique=True)
+    users = models.ManyToManyField(to=User)
+    def __str__(self):
+        return self.name
+
+class Punishment(models.Model):
```

Comment:
Maybe add an active property which is a booleanfield. An active punishment is one which is not paid. Might be usefull for leaderboard?